rust_src/src/cmds.rs

```diff
@@ -9,3 +9,21 @@ pub fn forward_point(n: LispObject) -> LispObject {
     let pt = ThreadState::current_buffer().pt();
     LispObject::from_fixnum(n.as_fixnum_or_error() + pt as EmacsInt)
 }
+
+//Move point to beginning of current line (in the logical order).
+//With argument N not nil or 1, move forward N - 1 lines first.
+//If point reaches the beginning or end of buffer, it stops there.
```

Comment:
Please use a triple-slash docstring, and use the full docstring that was in the C code.